server/modules/health-metrics/health-metrics.service.ts

```
@@ -0,0 +1,314 @@
+  }
+
+  // Get metrics history
+  async getMetricsHistory(options: MetricsHistoryOptions = {}) {
+    try {
+      const { hours = 24, interval = "hour" } = options;
+
+      let timeFilter: any = {};
+      let groupByInterval: any = {};
+
+      // Create time filter based on hours parameter
+      const hoursNum = parseInt(hours.toString()) || 24;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)